core/configuration/ignored_identifier.py

```diff
@@ -40,7 +40,7 @@ class IgnoredIdentifierConfiguration(ConfigurationTrait):
         format="narrow",
     )
 
-    def get_ignored_identifier_types(self) -> Set[str]:
+    def get_ignored_identifier_types(self) -> Any:
```

Comment:
Why is this a more generic type than Set[int] now?

Reply:
This method returns `self.ignored_identifier_types or tuple()` so it definitely can't be `Set[str]` as it is now due to the empty tuple, whatever `self.ignored_identifier_types` is. 
If I understood it correctly, `self.ignored_identifier_types` is descriptor `ConfigurationMetadata` whose `__get__` has `Any` as a [return type](https://github.com/ThePalaceProject/circulation/pull/629/files#diff-5e9d4d0fb44a1a5f3015f053632b819c1202eac7bc445f2662135cb40f7f033dR1116) so I used Any from that return type.

Reply:
Following the hierarchy of defined constants of the configuration it looks like each value in the `ALL_IDENTIFIER_TYPES` are strings.

Reply:
Yeah, this seems correct, thanks! Updated this and code from comment bellow in https://github.com/ThePalaceProject/circulation/pull/629/commits/0827f40d81efc2cbd485fed5a65ca09c36f116ec